flowmachine/flowmachine/core/query.py

```diff
@@ -112,7 +116,10 @@ def md5(self):
             self._md5 = md5(str(hashes).encode()).hexdigest()
             return self._md5
 
-    query_id = md5  # alias which is more meaningful to users than 'md5'
+    @property
+    def query_id(self):
```

Comment:
Why not just rename the md5 attribute?

Reply:
Hmm, I don't think this change is actually relevant to this PR. But yes, addressing the root issue seems more sensible than fixing a workaround.